libcloud/test/backup/test_dimensiondata.py

```diff
@@ -35,19 +34,13 @@ def setUp(self):
         self.driver = DimensionData(*DIMENSIONDATA_PARAMS)
 
     def test_invalid_region(self):
-        try:
+        with self.assertRaises(ValueError):
             self.driver = DimensionData(*DIMENSIONDATA_PARAMS, region='blah')
-        except ValueError:
-            pass
 
     def test_invalid_creds(self):
         DimensionDataMockHttp.type = 'UNAUTHORIZED'
-        try:
+        with self.assertRaises(InvalidCredsError):
```

Comment:
On a related note, where more general exceptions such as `ValueError` are thrown, it's also a good idea to assert on the exception message (if possible).

For example:

```python
expected_msg = 'Invalid region: blah'
self.assertRaisesRegexp(ValueError, expected_msg, DimensionData, *DIMENSIONDATA_PARAMS, region='blah')
```